lineage/figures/figure4.py

```diff
@@ -114,16 +114,16 @@ def figureMaker3(ax, x_Sim, output_Sim, x_Cen, output_Cen, xlabel="Number of Cel
 
     i += 1
     ax[i].axhline(y=100, ls='--', c='k', alpha=0.5)
-    sns.regplot(x="Cell number", y="State Assignment Accuracy", data=accuracy_sim_df, ax=ax[i], lowess=True, marker='+', scatter_kws=scatter_state_1_kws)
+    sns.regplot(x="Cell number", y="State Assignment Performance", data=accuracy_sim_df, ax=ax[i], lowess=True, marker='+', scatter_kws=scatter_state_1_kws)
     ax[i].set_xlabel(xlabel)
-    ax[i].set_ylim(bottom=0, top=101)
+    ax[i].set_ylim(bottom=50, top=101)
```

Comment:
Did the reviewers ask for these axis changes? I'm not generally a fan of y-axis that don't extend to the natural limit (in this case 0).

Reply:
Only for Bernoulli parameter estimation. Although the reviewer did ask for changing the yaxis if figure 4 (censored vs uncensored) but I will reverse that.